### PR TITLE
Refactors create collection form for digital

### PR DESCRIPTION
**Pull request by kaw2k**
This PR refactors the create collection screen for digital assets a bit to fix a few of our issues:

- If uploading a collection fails, you need to first clear it (which will trigger messages) and then re-select the collection
-  "decentrelized" is spelled wrong
- Using `alert` to communicate with the user when creating a collection is not great UX

~Note, I will go through the fungible flow as well after this PR lands or gets changed based on feedback.~

Both flows are now updated. 

https://github.com/aptos-labs/create-aptos-dapp/assets/694324/a90777d5-3fb5-4695-b894-ca5463d14b7a

~https://github.com/aptos-labs/create-aptos-dapp/assets/694324/78a03985-3005-4683-bced-18ca8d40e056~

### CODE REVIEW

**Comment by 0xmaayan** [comment]
we should probably want to exit the uploading process on first "cancel" on the wallet also

**Review by 0xmaayan — COMMENTED** [review]
Let's add all to the FA template also

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx [hunk 1]
```diff
@@ -219,215 +196,284 @@ export function CreateCollection() {
...
+              </>
+            }
+          />
         </div>
       </div>
     </>
   );
 }
+
+const CreateCollectionInput: FC<{
```
Comment (0xmaayan) [inline templates/digital-asset-template/frontend/pages/CreateCollection.tsx:361]: move all to a component folder?

**Comment by kaw2k** [comment]
I updated both flows, and both exit early on error.

**Review by 0xmaayan — APPROVED** [review]

**Pull request merged**